validate/automated/nodectlUpgrade.md

```diff
@@ -0,0 +1,712 @@
+---
+title: upgrading with nodectl
+hide_table_of_contents: false
+---
+
+import DocsCard from '@components/global/DocsCard';
+import DocsCards from '@components/global/DocsCards';
+import MacWindow from '@site/src/components/global/MacWindow';
+
+<head>
+  <title>MainNet 2.0 Automation with nodectl</title>
+  <meta
+    name="description"
+    content="MainNet 2.0 Automation - Upgrade Tessellation with nodectl"
+  />
+  <style>{`
+    :root {
+      --doc-item-container-width: 60rem;
+    }
+  `}
+  </style>
+</head>
+
+### Description
+
+This document will show you; through step-by-step instructions, how to upgrade your Node to the latest version of **Tessellation** or **nodectl**.
+
+**Tessellation** is the code name for the protocol that runs on your Node.  It is the guts of how your Node is able to operate on the **Hypergraph** or **Metagraph**.  
+
+**nodectl** is a utility that runs on your Node and helps you automate some of the more complex processes that Tessellation requires to run efficiently. 
+
+## Prerequisite
+### Installation
+
+This document assumes that you have nodectl with Tessellation already running on your Node.  
+
+If this is not the case, please review the [installation guide](./nodectl.md) `Getting Started`: to determine the type of installation required.
+
+## Upgrade
+
+<DocsCards>
+  <DocsCard header="Upgrade nodectl" href="#upgrading-with-nodectl" img="/img/home/state-channel.jpg">
+    <p>Upgrade the nodectl utility.</p>
+  </DocsCard>
+
+  <DocsCard header="Upgrade Tessellation" href="#upgrade-tessellation" img="/img/home/community.jpg">
+    <p>Upgrade Tessellation using nodectl.</p>
+  </DocsCard>
+</DocsCards>
+
+## Upgrading with nodectl
+
+There are `3` upgrade mechanisms regarding `nodectl`.
+1. Upgrade the **nodectl** utility to a new version. (binary)
+2. Upgrade your Node to handle new features of **nodectl**, or modify some elements to make your Node more efficient when working with the **nodectl** utility.
+3. Upgrade **Tessellation** and any elements of the VPS that may require modification to work with the **Tessellation** protocol.
+
+Mechanisms `2` and `3` may include updating features of the VPS to facilitate functionality of your Node.
+
+Using **nodectl's** built-in [upgrade command](./nodectlCommands#upgrade_nodectl), you do not have to worry about the differences mentioned when upgrading nodectl.  **nodectl** is smart enough to guide you through the upgrade process.  It will know if you need any extra steps to complete. 
+
+These steps include:
+  - Is it safe to upgrade from the current version of **nodectl** your Node is running to the latest version?  
+    - There is an [upgrade path](./nodectlUpgradePath) that might need to be taken; dependent on, how long you have waited to upgrade your Node.
+  - Does your Node require any extra upgrade steps to complete the **nodectl** upgrade for effective operation.
+
+:::caution 
+Currently there is not a scheduled release process for nodectl.
+:::
```

Comment:
I'd suggest removing this. There's no reason to call out that we don't use a certain process.

Reply:
removed